lib/bar/BottomBar.js

```diff
@@ -111,19 +112,27 @@ export default class BottomBar extends React.Component {
   }
 
   _renderActionSheet() {
-    const { displayActionButton, onAction } = this.props;
+    const { customButton, displayActionButton, onAction } = this.props;
+
+    let components = [];
```

Comment:
Let's make this `const components = []` as the reference it holds doesn't change.